docs/specs/bug_and_feedback_submission.md

```diff
@@ -0,0 +1,88 @@
+
+Cons:
+- No support for images
+- Cumbersome marshalling from structured data to query string in Suite and then from query string to CSV in Keboola. Some layers of the setup [might also limit](https://stackoverflow.com/a/812962/5698865) length of the query string rendering it unsuitable to transmit larger sets of data.
+- `GET` requests and query strings are not supported by the [Beacon API](https://developer.mozilla.org/en-US/docs/Web/API/Beacon_API) which will be needed for analytics improvements in the future.
+- It's brittle. The CloudFronts and buckets are configured manually in AWS with limited access and no versioning.
+
+## Solution
+For the first iteration we'll use the same setup as with Analytics. We'll setup four new paths on the `data.trezor.io` domain:
+- `/suite/feedback/devel`
+- `/suite/feedback/prod`
+- `/suite/bugs/devel`
+- `/suite/bugs/prod`
+
+with the same exact logging configuration. Tomas ferko will re-use the python scripts in Keboola. 
+
+If user includes an image in a message it will be uploaded separately and the message will contain just a textual reference to the image. We will provision a pair (for production and dev/staging) of S3 buckets to store the images along with an IAM user who can [only `putObject`](https://stackoverflow.com/questions/37739086/how-to-allow-only-putobject-permissions-on-specific-directory-in-amazon-s3-bucke/37754260#37754260) to these buckets. This prevents unauthorized listing and reading of the uploaded images.
+
+When putting an object into a bucket [the client must give it a key](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingMetadata.html). We'll use an UUID implementation. This prevents both accidental and malicious overriding of uploaded images given the UUID is sufficiently random and has sufficiently large value space. The generated UUID will be used in the actual message as reference of the image.
+
+This opens a possible attack vector for incurring large bills to SatoshiLabs by an attacker uploading large amounts of data to the buckets. This can't be 100% prevented as the API must remain open because Suite's users don't authenticate in any way. A rate limiting could prevent this but that would require some kind of application layer in between the clients and the S3. Instead, for start, we'll start by setting up a [CloudWatch alarm](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html) that will warn us if daily uploads exceed some threshold.
+
+## Data
+This section describes what data are sent in the query string. We start with smaller set of fields that can all fit into one level of nesting so that we don't have to decide [which encoding convention](https://stackoverflow.com/a/9547490/5698865) to use for nested fields.
+
+### Bugs
+- `description` The textual description of the bug written by the user.
+- `path` Current url path. Corresponds to the "category" in the Designs.
```

Comment:
This field awaits a clarification in the product spec. (There's one open comment I'd like to link but Notion doesn't provide permalinks for comments. ./ )

Reply:
Matej specified a static list of few categories. We'll send these as a string. In the app we'll write a function that will map routes to categories to support the `current screen` feature in the bug submission.

This spec edited [here](https://github.com/trezor/trezor-suite/pull/3764/commits/6a4ee408c230c7211083977eb51d17475f7df279).

Resolving.